src/components/MultiSelect/MultiSelect.js

```diff
@@ -60,6 +60,11 @@ export default class MultiSelect extends React.Component {
      */
     type: PropTypes.oneOf(['default', 'inline']),
 
+    /**
+     *  Specify title to show title on hover
+     */
+    title: PropTypes.bool,
```

Comment:
I naïvely imagined prop named `title` is simply propagated as `title` attribute, which does not seem to be the case. Probably `useTitleInItem` or a like?

Reply:
I see the confusion! I will change the prop name to `useTitleInItem`